lib/postprocessing/image.js

```diff
@@ -459,6 +469,35 @@ async function imagePostProcess(intermediateRendition, rendition, directories) {
     await write(img, rendition.path);
 }
 
+async function convertSvg(path, instructions) {
+
+    // IM automatically keeps aspect ratio
+    const width = instructions.width || 400;
+    const height = instructions.height || 400;
```

Comment:
one question, why is the default 400?

Reply:
This was on another thread about what does AEM default to which was 400x400
Super long thread but it starts around here https://cq-dev.slack.com/archives/GAMJLHSBH/p1603232882448300?thread_ts=1603227975.430400&cid=GAMJLHSBH